registration/registration-services/src/main/java/io/mosip/registration/service/packet/impl/PacketSynchServiceImpl.java

```diff
@@ -281,6 +281,13 @@ private List<SyncRegistrationDTO> getPacketSyncDtoList(@NonNull List<Registratio
 
 			SyncRegistrationDTO syncDto = new SyncRegistrationDTO();
 			syncDto.setRegistrationId(registration.getId());
+			//To Do

```

Comment:
This is not present in latest code,Please take latest code from upstream

Reply:
These changes are not present in tf_nira_dev,We cant take these to tf_nira_dev branch again